core/dbt/graph/selector_methods.py

```diff
@@ -326,7 +326,11 @@ class PathSelectorMethod(SelectorMethod):
     def search(self, included_nodes: Set[UniqueId], selector: str) -> Iterator[UniqueId]:
         """Yields nodes from included that match the given path."""
         # get project root from contextvar
-        root = Path(cv_project_root.get())
+        project_root = get_project_root()
+        if project_root:
+            root = Path(project_root)
+        else:
+            root = Path.cwd()
```

Comment:
would it be possible / more appropriate to use [`default_project_dir`](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/cli/resolvers.py#L6) here?

Reply:
It's basically copied directly from project.project_root. So at this point it would be more surprising to me to have the name different. I'm not a big fan of that name, but since this is going to be backported I'd prefer to leave it for now and leave the rename for some technical debt cleanup.